frontend/app/abtests/ABTest.scala

```diff
@@ -0,0 +1,66 @@
+package abtests
+
+import java.time.Duration.ofDays
+
+import abtests.AudienceId.idFor
+import play.api.mvc.{Cookie, Request, RequestHeader}
+
+
+case class Allocation(test: ABTest, variant: String) {
```

Comment:
Good spot, removed with d79c40a - this was left over from the code I copied from Contributions.